src/hardware/motor.cpp

```diff
@@ -228,6 +231,15 @@ float StepperMotor::getMicrostepMultiplier() const {
     return (this -> microstepMultiplier);
 }
 
+void StepperMotor::simpleStep() {
```

Comment:
What is the purpose of this function? It doesn't seem to do anything that the regular step routine couldn't handle

Reply:
I use it for tests. Please leave it for a while.

Reply:
Will do